src/main/resources/component-biz.conf

```diff
@@ -341,6 +342,9 @@ timer.daily {
     # Determines when analytical tasks are scheduled...
     analytical-engine = 23
 
+    # Determines when expires process logs are removed...
+    cleanup-processes = 3
```

Comment:
This collides with the hour ES Dumps are created in OXOMI. Would changing this to 2 or 4 be okay? Else we would have to remeber changing this in OX when upgrading the biz version